src/sql/SqlService.ts

```diff
@@ -42,6 +42,92 @@ import {HzLocalTime, HzLocalDate, HzLocalDateTime, HzOffsetDateTime} from './Dat
 
 /**
  * SQL Service of the client. You can use this service to execute SQL queries.
+ *
+ * The service is in beta state. Behavior and API might change in future releases. Binary compatibility is not
+ * guaranteed between minor or patch releases.
+ *
+ * If this cluster is a Hazelcast Jet cluster, a statement can be executed by either the default SQL backend or by Hazelcast
+ * Jet backend, as a Jet job. If some of the features used in a statement isn't supported by the default backend, the engine will
+ * attempt to execute it using Jet. This class is the API to both backends.
+ *
+ * The text below summarizes features supported by the default SQL engine. For a summary of Hazelcast Jet SQL features
+ * see `com.hazelcast.jet.sql` package javadoc in Hazelcast Jet (once released).
+ *
+ *
+ * ### Overview
+ * Hazelcast is able to execute distributed SQL queries over the following entities:
+ * * IMap
+ *
+ * When an SQL statement is submitted to a member, it is parsed and optimized by the `hazelcast-sql` module, that is based
```

Comment:
I don't think we should mention Calcite etc. in the clients. It is an implementation detail. Also, I have modified the documentation in the Python client PR so that we mention these details at the end, instead of scaring the user at the beginning with Java terms like classpath, etc. Maybe we can do the same here

Reply:
I vote for removing it